reciprocalspaceship/dataset.py

```diff
@@ -878,33 +878,44 @@ def compute_multiplicity(self, inplace=False, include_centering=True):
         return self
 
     @inplace
-    def assign_resolution_bins(self, bins=20, inplace=False, return_labels=True):
+    def assign_resolution_bins(
+        self, bins=20, inplace=False, return_labels=True, return_edges=False
+    ):
         """
         Assign reflections in DataSet to resolution bins.
 
         Parameters
         ----------
         bins : int
-            Number of bins
+            Number of bins (default: 20)
         inplace : bool
-            Whether to add the column in place or return a copy
+            Whether to add the column in place or return a copy (default: False)
         return_labels : bool
             Whether to return a list of labels corresponding to the edges
-            of each resolution bin
+            of each resolution bin (default: True)
+        return_edges : bool
+            Whether to return bin edges that define the resolution bin boundaries
```

Comment:
I always find it hard to infer what "bin edges" means. For instance, is it an bins X 2 array or array with  bins + 1 entries? This needs to be clearer.

Reply:
I'll specify that it's a 1d array with bins+1 entries